tools/ScenarioTest.ResourceManager/ScenarioTest.ResourceManager.csproj

```diff
@@ -39,4 +39,8 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Microsoft.Azure.Management.ResourceManager" Version="3.8.0-preview" />
```

Comment:
This project already has a dependency on `Microsoft.Azure.Management.ResourceManager`, see https://github.com/Azure/azure-powershell/blob/master/tools/Common.Netcore.Dependencies.Test.targets#L10
so this is duplciated